src/v/redpanda/application.cc

```diff
@@ -970,6 +970,16 @@ void application::admin_register_security_routes(ss::http_server& server) {
                   ss::json::json_return_type(ss::json::json_void()));
             });
       });
+
+    ss::httpd::security_json::list_users.set(
+      server._routes, [this](std::unique_ptr<ss::httpd::request>) {
+          std::vector<ss::sstring> users;
+          for (const auto& it : controller->get_credential_store().local()) {
```

Comment:
nit: maybe using `const auto& [key, value] :.. ` would make it more readable.

Reply:
fixed